wiki/Manual.md

```diff
@@ -6138,7 +6140,7 @@ In case you experience any difficulty connecting to a particular exchange, do th
 - Check if there were any news from the exchange recently regarding downtime for maintenance. Some exchanges go offline for updates regularly (like once a week).
 - Make sure that your system time in sync with the rest of the world's clocks since otherwise you may get invalid nonce errors.
 
-## Notes
+Further notes: 
```

Comment:
I think we should highlight this title otherwise it's hard to see

Reply:
I wanted to de-hyperlink that (so it was not hashtaged under `#notes` url), would you might to edit this file on the fly and commit per you think it to be better? i.e. bolded, or reverted back into `##` title?